data/json/npcs/island_prison/prisoners.json

```diff
@@ -0,0 +1,513 @@
+          "and": [
+            { "u_has_var": "talked_to_leader", "type": "dialogue", "context": "first_meeting", "value": "yes" },
+            {
+              "not": { "u_has_var": "prisoner_leader_mission", "type": "mission", "context": "accepted", "value": "yes" }
+            },
+            {
+              "not": { "u_has_var": "prisoner_leader_mission", "type": "mission", "context": "completed", "value": "yes" }
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "id": "TALK_PRISONER_LEADER_LEARNED_ABOUT_CANNIBAL",
+    "type": "talk_topic",
+    "dynamic_line": [
+      "Huh, even so?  I'm not saying I approve your… tastes, but I definitely respect people who use all possibilities life has to offer."
+    ],
+    "speaker_effect": { "effect": { "u_add_var": "talked_to_leader", "type": "dialogue", "context": "cannibal", "value": "yes" } },
+    "responses": [ { "text": "Yeah, we share the same opinions on this matter.", "topic": "TALK_NONE" } ]
+  },
+  {
+    "id": "TALK_PRISONER_LEADER_INQUIRY",
+    "type": "talk_topic",
+    "dynamic_line": [
+      "Well, that's a boastful statement.  Not that I don't believe it, it's just….  Ah, nevermind.  Let's get to the point.  What do you want?"
+    ],
+    "responses": [
+      {
+        "text": "I want to get off this <swear> prison.  I hoped you could help me with that.",
+        "topic": "TALK_PRISONER_LEADER_GIVES_WORK"
+      }
+    ]
+  },
+  {
+    "id": "TALK_PRISONER_LEADER_FROWN",
+    "type": "talk_topic",
+    "dynamic_line": [ "Hold the horses, pal.  Nobody allowed you to stay here - yet." ],
+    "responses": [
+      {
+        "text": "Please!  I don't want to get back there!  I'm tired and hungry!  I just need time to rest!",
+        "topic": "TALK_PRISONER_LEADER_ANNOYED"
+      },
+      { "text": "Shit!  Looks like I'm on my own now.  Thanks for nothing.  Bye.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_PRISONER_LEADER_ANNOYED",
+    "type": "talk_topic",
+    "dynamic_line": [
+      "We don't have much space, and there's not much food.  Besides, I don't know you at all, and something tells me you won't survive another day."
+    ],
+    "responses": [
+      {
+        "text": "[STR 11] I'm tougher than it seems!",
+        "topic": "TALK_PRISONER_LEADER_NO",
+        "condition": { "u_has_strength": 11 }
+      },
+      {
+        "text": "[Survival 2] I have some survival expertise!",
+        "topic": "TALK_PRISONER_LEADER_NO",
+        "condition": { "u_has_skill": { "skill": "survival", "level": 2 } }
+      },
+      { "text": "I guess I need to find some other place…", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_PRISONER_LEADER_NO",
+    "type": "talk_topic",
+    "dynamic_line": [
+      "The answer is still no.  In fact it might be better for us to kill you right away, as, you know, the less mouths to feed, the better.  But I'm in good mood today, so I decided to help you get out of this island."
```

Comment:
```suggestion
      "The answer is still no.  In fact it might be better for us to kill you right away - you know, the less mouths to feed, the better.  But I'm in a good mood today, so I've decided to help you get out of this island."
```